### PR TITLE
mimsmind functions, numbered lines and seventeen1

### PR DESCRIPTION
**Pull request by Neeragrover**

### CODE REVIEW

File: numbered_lines.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import random 
+
+def main():   # DO NOT CHANGE BELOW
```
Comment (danielsgriffin) [inline numbered_lines.py]: Good job.

File: numbered_lines.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import random
```
Comment (danielsgriffin) [inline numbered_lines.py]: Random, eh?

File: seventeen1.py [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+    if input in correct_input:
+        return 1
+    else:
+        return -1
+        
+def check_for_num(number):
+	""" This function checks if the value entered by the user is numeric"""
+	try:
+		int_num = int(number)
+	except:
```
Comment (danielsgriffin) [inline seventeen1.py]: Good, but
We want to start catching specific errors:
```python
except ValueError:
```

File: seventeen1.py [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+            print "\nComputer's turn..."
+            remaining_after_comp = computers_turn(remaining_after_human,user_input)
+          
+        num_of_marbels = remaining_after_comp
+        print "Number of marbels left in jar",num_of_marbels  
+    
+    print "You won" 
+        
+         
+if __name__ == '__main__':
```
Comment (danielsgriffin) [inline seventeen1.py]: Good job.

minor note: try to have the spacing match the example (for readability and checking word):
you had:
```
Computer's turn...
Computer removed 1 marble.
Number of marbels left in jar 1
Your turn: How many marbles will you remove (1-3)?3
Sorry, that is not a valid option. Try again!.
```

**Comment by danielsgriffin** [comment]
Good job.

File: seventeen1.py (hunk 3 above)
Comment (Neeragrover) [inline seventeen1.py]: but wont the except throw the user out of the while loop> didn't we want to keep going ?

**Pull request closed without merging**